content/docs/04-clusters/03-edge/09-site-deployment/05-site-installation/06-edge-host-registration.md

```diff
@@ -34,9 +34,11 @@ Select the registration method that best fits your organizational needs and revi
 
 ## Auto Registration
 
-You can automate the registration process by using registration tokens. This method requires providing the registration token in the user data. If a default project was selected for the registration token, that is the project the Edge host will be registered under. You can override the default project by providing a project ID in the user data.
-
-
+Use the mandatory auto-registration tokens to automate the registration process by providing the registration token in the user data. The device registers and appears on the **Edge** hosts page when the `autoRegister` attribute is true.
+**Note**: Providing the auto-registration token for all **Edge** host registration methods is mandatory.  
+The **Edge** host will be registered under the default project selected for the registration token. You can override the default project by providing a project ID in the user data. 
+**Note**: It is mandatory to assign a project when creating a registration token to complete the auto-registration process.
```

Comment:
We don't use **NOTE**, remove those. Additional information can go in an info box. Overall, this paragraph needs cleanup to help the user understand the behavior of the new variable. The token is mandatory when using this method. So be careful how you use the mandatory term otherwise it will cause confusion.

Reply:
One question Karl, You mentioned the token is mandatory when using this method. I have one question here
in this ticket : [PE-1602](https://spectrocloud.atlassian.net/browse/PE-1602)

Suppose the registration token is required. What happens with the other two registration methods (manual, and QR code)?

Users can add the device in UI, but the edge host agent will not connect to Saas until the registration token is provided in userdata.

Does this mean the token is mandatory for all edge host registration methods? 
If you could please clarify this for me.